nss/integration-tests/helper_test.go

```diff
@@ -37,7 +37,7 @@ func buildRustNSSLib(t *testing.T) (libPath string, rustCovEnv []string) {
 	out, err := cmd.CombinedOutput()
 	require.NoError(t, err, "Setup: could not build Rust NSS library: %s", out)
 
-	// When building the crate with dh-cargo, this env is set to indicate which arquitecture the code
```

Comment:
Oops... I accidentally created a hybrid between Brazilian Portuguese and English :sweat_smile: